### ISSUE TITLE
GameGuru Max Crashing "RunTime Error 7018"

### ISSUE THREAD

**Issue by TheCzar1** [TheGameCreators/GameGuruRepo#2835]
I recently installed GameGuru Max and every time I try, and run one of the demo levels, the program brings up a "runtime Error 7018" message, and then when I click "ok"; the whole program shuts down! I have tried uninstalling and reinstalling GameGuru Max, but I am having the same problem still, and I do not know what to do?

My system specs:
Intel Core i9 12900KS
Asus ROG Maximus Apex Z690
G.Skill 32 GB DDR5 6400 MHZ
Asrock Radeon RX 6900 XT Phantom OC
Windows 11 Pro

**Comment by rvanner** [comment]
@LeeBamberTGC I also have a user in tech support reporting similar;

Can't run or edit any of the demo games. New install of latest version. Intel I9, win 11 (unfortunately). Have plenty of ram. Always the same error.
Processor 12th Gen Intel(R) Core(TM) i9-12900KF, 3187 Mhz, 16 Core(s), 24 Logical Processor(s)
Windows 11
BIOS Version/Date American Megatrends International, LLC. A.A2, 11/3/2021
Installed Physical Memory (RAM) 64.0 GB
Nvidia GeForce RTX 3080 Ti

**Comment by LeeBamberTGC** [comment]
@rvanner I see this error when the install is corrupt with missing files. Running the Steam VERIFY FILES will scan the whole folder and replace any missing files. The above assumes the software runs okay but hits a problem when a demo project is loaded/run.

**Comment by TheCzar1** [comment]
I do not have the Steam version of GameGuru Max; I have the standalone version, and I have tried a few more times of uninstalling the program and reinstalling it on different drives in my computer, but I still keep getting this same "Runtime Error 7018", and the program always shuts down once this error pops up, and I hit "ok"!

**Comment by LeeBamberTGC** [comment]
@TheCzar1 It could be an old updater. When you do a re-install grab the latest Tiny Installer from the TGC product page. It has been updated for release to handle the demo files and make sure you have all the files you need as part of the core download.

**Comment by TheCzar1** [comment]
I have always tried and re-downloaded from TGC webaite with the Tiny installer and everytime; I still get thay error, but I will try it again, and see with the new update if it works!

**Comment by TheCzar1** [comment]
I tried once again to re-install GameGuru Max with the Tiny Installer from the products page on TGC site, and I am still getting the same error "Runetime Error 7018 (loading_main)". I hit "ok" and again the whole program shuts down and exits out.

**Comment by TheCzar1** [comment]
I should also not the Tiny installer I used was from my product page when I log into my account. Is the right one?

**Comment by rvanner** [comment]
@TheCzar1 Yes that's the correct one. Would you like to try the Steam version? If so, email me at rick@thegamecreators.com and I'll sort you out with a Steam copy. We can then see if it still happens or if that fixes it for you.

**Comment by TheCzar1** [comment]
Yes let me try that version out and see if it fixes the issue.

**Comment by rvanner** [comment]
OK, please email me direct

**Comment by SwatGuy13** [comment]
I'm having the same issue. Did the install with the steam client work?

**Comment by TheCzar1** [comment]
@SwatGuy13 Yes the Steam version worked perfect with no issues! I could play all the demo levels with no problem. It seems the Steam version is the way to go to resolve this "runtime error 7018"!

**Comment by rvanner** [comment]
@SwatGuy13 Please contact me and I'll set you up with a Steam version.

**Comment by SwatGuy13** [comment]
That's ok I got a key and D/L last night. It's working fine. If I have any issues I will let you know.
Cheers.

**Comment by Kravenwolf** [comment]
@rvanner, I am also getting the same error when either loading a demo level or trying to add an asset to a new map. I sent you an email to get a Steam Key. 

Only thing I did different this time to get the error (didn't get it the last time I used GGM several months ago) was I installed GGM on my HDD, instead of my C/OS drive, because my SSD was getting full. And I only installed the 2 options ticked by default this time around in the installer. (15~GB)
